source/db/de-projects.json

```diff
@@ -3302,7 +3302,7 @@
   },
   {
     "development_stage": "released",
-    "description": "Telefonnummer basierter Messagingclient für die Masse, unter Verwendung von XMPP und OpenPGP Verschlüsselung. Unterstützt durch Communitybetrieben Servern.",
+    "description": "Massentauglicher, Telefonnummer basierter Sofortnachrichtenclient, unter Verwendung von XMPP und OpenPGP-Verschlüsselung. Unterstützt durch von der Gemeinschaft betriebene Server.",
```

Comment:
Servers can driven by everyone (by the community)

So I think it's correct ad I said.

Die Server können von jedem (also von der Gemeinschaft nicht nur für die Gemeinschaft) betrieben werden.

Deshalb glaube ich dass meine Formulierung richtig ist.

Gemeinnützig sagt m.E. nur aus das durch den Betrieb keine Gewinne erzielt werden.